src/commcare_cloud/commcare_cloud.py

```diff
@@ -148,9 +148,67 @@ def make_parser(available_envs, formatter_class=RawTextHelpFormatter,
     return parser, subparsers, commands
 
 
+def make_changelog_parser():
```

Comment:
This name is confusing. `make_command_parser` returns an argument parser whereas this function compiles the changelog.

I'd rename this to something like `compile_changelog`